### ISSUE TITLE
Close images after pasting

### ISSUE THREAD

**Issue by zmwangx** [zmwangx/storyboard#19]
Several memory leaks warnings or something along that line always show up in `unittest` results when running `python3 setup.py test`. There's no such warning when I run `python2 setup.py test`. Might be different logging levels, or something different in Pillow's internals. The issue may lie in Pillow's internals, but it could also be because I leave dangling images around after pasting. (They will eventually get garbage collected; not sure why the warnings.) Anyway, I'll close pasted images from now on.

**Comment by zmwangx** [comment]
Implemented in `storyboard.storyboard.tile_images` in c87733a7fcd4b44174b7c246ad34e84dbf7a5114. Completely fixed in d7405f7463c569e85b5da43739224638c6b3bc26.

**Comment by zmwangx** [comment]
On closer inspection, the `ResourceWarning`s weren't due to dangling images in the first place. They were due to not closing the pipes after calling `terminate()` on a `subprocess.Popen` object, i.e., after `p.terminate()`, `p.communicate()` is needed for cleaning up.

I noticed this after reading the [3.5 nightly warnings](https://travis-ci.org/zmwangx/storyboard/jobs/59183116) on Travis. 3.4 warnings are unhelpful, raised inside `subprocess.py` — well, who the hell knows who called subprocess, an external library or what? 3.5 is really help in pinning down 

``` python
/home/travis/build/zmwangx/storyboard/src/storyboard/metadata.py:258: ResourceWarning: unclosed file <_io.BufferedReader name=8>
  self.scan_type = self._get_scan_type(ffprobe_bin, print_progress)
...
```

I inspected `_get_scan_type` and realized `terminate()` is the culprit. Now fixed in 31c9d56. Thanks 3.5!
